references/cli/adopt.go

```diff
@@ -188,6 +190,20 @@ func (opt *AdoptOptions) Complete(f velacmd.Factory, cmd *cobra.Command, args []
 	} else {
 		opt.AdoptTemplate = defaultAdoptTemplate
 	}
+	if opt.AppName != "" {
+		var ctx = context.Background()
+		app := &v1beta1.Application{}
+		err := f.Client().Get(ctx, apitypes.NamespacedName{Namespace: opt.AppNamespace, Name: opt.AppName}, app)
+		if err == nil && app != nil {
+			if !opt.Yes {
+				userInput := NewUserInput()
+				confirm := userInput.AskBool("Application '%s' already exists, apply will override the existing app with the adopted one, please confirm [Y/n]: "+opt.AppName, &UserInputOptions{AssumeYes: true})
```

Comment:
What does `AssumeYes:true` mean?

Reply:
My bad! It should be false as it means if the user does not provide input, the default value will be considered as true.